src/components/navBar/index.tsx

```diff
@@ -1,17 +1,67 @@
 import Image from 'next/image';
+import Link from 'next/link';
+import { LOGIN_URL, USER_PROFILE_URL, DEFAULT_AVATAR } from '@/components/constants/url'
+import useAuthenticated from '@/hooks/useAuthenticated';
 import styles from '@/components/navBar/navBar.module.scss';
-
 const RDSLogo = '/RDSLogo.png';
+const GITHUB_LOGO = '/github-white.png';
+
+const NavBar = () => {
+  const { userData, isLoggedIn } = useAuthenticated()
 
-const NavBar = () => (
-  <nav className={styles.navBar}>
-    <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
-    <a href="https://welcome.realdevsquad.com/">Welcome</a>
-    <a href="https://www.realdevsquad.com/events.html">Events</a>
-    <a href="https://members.realdevsquad.com/">Members</a>
-    <a href="https://crypto.realdevsquad.com/">Crypto</a>
-    <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
-  </nav>
-);
+  return (
+    <nav className={styles.navBar}>
+      <div>
+        <a className={styles.logo} href="https://realdevsquad.com">
```

Comment:
Could we look into fetching links from constant files?

Reply:
done